resources/build/l10n/README.md

```diff
@@ -22,7 +31,7 @@ To check your CLI setup,
 crowdin list project
 ```
 
-You should see the CLI fetching project info and generating a list of files associated with the project.
+You should see the CLI fetching project info and generating a list of files associated with the project. If you get Java ConnectionExceptions, try connecting VPN.
```

Comment:
Which VPN would this be?

Reply:
I was consistently bogged by connection exceptions. But when I (Express VPN) to Los Angeles, things work for me. ymmv

Reply:
Okay. Can we drop the VPN note from the instructions because it will be confusing as it is! Maybe add some note and background into our wiki somewhere...